### DISCUSSION TITLE
How do I change its default shell？

### DISCUSSION THREAD

**Discussion by wakeup-go** [Ulauncher/Ulauncher#1029] [Q&A]
Ulauncher shell is ugly, I want to change to a beautified shell, such as Kali's Zshshell, what should I do?

**Comment by friday** [comment]
Ulauncher doesn't have a shell. There's no built-in Command prompt.

If you're talking about theming, you can find that [here](http://docs.ulauncher.io/en/latest/themes/themes.html)
